xml/docu_styleguide.structure.xml

```diff
@@ -1516,8 +1516,8 @@ xml:id="tab-install-source"</screen>
    <listitem>
     <para>
      If all list items are nouns only, do not capitalize their first letter.
-     Use sentence-style capitalization for list items that are full sentences.
-     Use title-style capitalization for terms in descriptive lists.
+     Use sentence-style capitalization for list items that are full sentences,
```

Comment:
Why the comma?--This is a list of just two items.

Reply:
That was Liam's add. I don't know. I'm removing it.